_shows/16_17/infectious.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Infectious
+playwright: Emma White
+student_written: true
+season: Fringe
+season_sort: 250
+period: Spring
+venue: New Theatre
+date_start: 2017-03-13
+date_end: 2017-03-14
+
+cast:
+- role: Jess
+  name: Felicity Chilver
+- role: Abbie
+  name: Daniella Finch
+
+crew:
+- role: Director
+  name: Emma White
+- role: Producer
+  name: Emily Sterling
+- role: Design Assistant
+  name: Emily Wong
+- role: Lighting Designer
+  name: Ben Woodford
+- role: Technical Director
+  name: Joanne Blunt
+- role: Poster Designer
+  name: Max Miller
+
+prod_shots: hdzVLb
```

Comment:
That all depends on how the cache behaves. A PR shouldn't pollute the master cache, but if accidentally merged (a PR is a request to 'pull' changed into master), it will.